languages/csharp/exercises/concept/enums/EnumsTests.cs

```diff
@@ -2,35 +2,59 @@
 
 public class LogLineTests
 {
-    [Fact]
-    public void ParseError() =>
-        Assert.Equal(LogLevel.Error, LogLine.ParseLogLevel("[ERROR]: Disk full"));
+    [Fact(Skip = "Remove this Skip property to run this test")]
```

Comment:
The very first test should not be skipped by default:

```suggestion
    [Fact]
```